src/pages/explorer/index.tsx

```diff
@@ -9,9 +9,27 @@ import { StrategyProvider, useStrategyCtx } from 'hooks/useStrategies';
 import { ExplorerTabs } from 'components/explorer/ExplorerTabs';
 import { useEffect, useState } from 'react';
 import { explorerEvents } from 'services/events/explorerEvents';
+import { toPairSlug } from 'utils/pairSearch';
+import config from 'config';
 
 export const ExplorerPage = () => {
   const { slug, type } = useExplorerParams();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (slug || type === 'wallet') return;
```

Comment:
I think it's more readable like that. Also if we add a type in the future this would be the correct logic.
```suggestion
    if (slug || type !== 'token-pair') return;
```

Reply:
Agreed